stages/org.osbuild.firewall

```diff
@@ -92,6 +119,35 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_item in zones:
+        # specifying an empty zone flag results in the source being applied to
+        # the default zone
+        zone_name = zone_item['name']
+        # check that the given zone exists, if not create it
+        if zone_name != "":
+            res = subprocess.run(["chroot",
+                                  tree,
+                                  "firewall-offline-cmd",
+                                  f"--info-zone={zone_name}"],
+                                 check=False)
+            # INVALID_ZONE error code
+            if res == 112:
```

Comment:
`res` here is a `CompletedProcess(args=['chroot', '/run/osbuild/tree', 'firewall-offline-cmd', '--info-zone=new_zone'], returncode=112)`, so we need `res.returncode`. Same below.